### ISSUE TITLE
[Farm Pokestops] Updating Pokestops...

### ISSUE THREAD

**Issue by dragonite00123** [finndev/PokeBuddy#1083]
What is the cause of this? This is a PTC account.

**Comment by haux1231** [comment]
Permanent ban.

**Comment by dragonite00123** [comment]
Proof that there is already a perma ban?

**Comment by haux1231** [comment]
I am having the same problem. Also, I cant log in to the game anymore, Getting the failed to get game data from the server.
I tried using another account on Pokebuddy and it worked just fine.

**Comment by haux1231** [comment]
I can be wrong tho. I hope I am wrong but all I can figure out at the moment is that I've just been banned permanently

**Comment by majdak79** [comment]
so many cheated accounts could have been saved, when you would stop yesterday guys. think about that.

**Comment by Deuxgod** [comment]
Right. Like there's accounts that haven't been logged for 2 weeks and still got banned.

**Comment by majdak79** [comment]
cheaters accounts have to be banned. i agree with that. save accounts stop cheating guys. you are also destroying game for others.

**Comment by haux1231** [comment]
Well, I aint capturing any gym or battling them. I am just collecting the Pokemons. I battle the gym with my main account(the one that is legit) @majdak79
